src/memray/reporters/tui.py

```diff
@@ -290,6 +290,7 @@ class AllocationTable(Widget):
     sort_column_id = reactive(default_sort_column_id)
     snapshot = reactive(_EMPTY_SNAPSHOT)
     current_thread = reactive(0)
+    merge_threads = False
```

Comment:
Hm, I would think we'd want this to be a reactive attribute, since we need to repopulate the table whenever it changes.